server/.style.yapf

```diff
@@ -0,0 +1,2 @@
+[style]
```

Comment:
We have previously decided on a 120 character column size.  Is the revert to 80 characters intentional?  I'd rather not change this unless there is a good reason (the argument for 120 is that it matches the working style & large format editors most of us work in today).

Reply:
I missed this. I've adjusted it back to 120.